_posts/2021-07-06-Mozfest-2021-Open-Source-Design-helps-out-Simply-Secure-with-a-UX-Clinic.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title: "Mozfest 2021: Open Source Design helps out Simply Secure with a UX Clinic"
+date: 2021-08-06
+categories: design open source 
+author: [Eriol Fox]() & [Core Open Source Design team]()
```

Comment:
@Erioldoesdesign this will cause issues again. Need to just be "Eriol Fox & Core Open Source Design team"